staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -484,3 +490,190 @@ func TestCacheWatcherStoppedInAnotherGoroutine(t *testing.T) {
 		w.Stop()
 	}
 }
+
+func TestTimeBucketWatchersBasic(t *testing.T) {
+	filter := func(_ string, _ labels.Set, field fields.Set) bool {
+		return field["spec.nodeName"] == "host"
```

Comment:
this filter is not used anywhere - so please make it a trivial one (always return true) - this is unnecessary complication

Reply:
done